consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -0,0 +1,73 @@
+package streams.procedures
+
+import org.apache.commons.lang3.StringUtils
+import org.neo4j.logging.Log
+import org.neo4j.procedure.*
+import streams.StreamsEventConsumerFactory
+import streams.StreamsEventSinkConfigMapper
+import streams.StreamsSinkConfiguration
+import java.util.stream.Stream
+
+class StreamResult(@JvmField val event: Any)
+
+class StreamsSinkProcedures {
+
+    @JvmField @Context
+    var log: Log? = null
```

Comment:
can we make this lateinit?

Reply:
Unfortunately `@JvmField` (which is required to expose de field as _Java public_) is incompatible with `lateinit`